src/views/Home/components/Banners/FarmAuctionsBanner.tsx

```diff
@@ -82,12 +89,39 @@ const RightWrapper = styled.div`
 
 const FarmAuctionsBanner = () => {
   const { t } = useTranslation()
+  const currentBlock = useCurrentBlock()
 
-  return (
+  const farmAuctionContract = useFarmAuctionContract(false)
+
+  const { data: currentAuction = null } = useSWRImmutable(['farmAuction', 'currentAuction'], async () => {
+    const auctionId = await farmAuctionContract.currentAuctionId()
+    const [auctionData, contenders] = await multicallv2(
```

Comment:
could we use `useSWRMulticall`  in this case ?

Ideally, `multicallv2` shouldn't be used in the Component-level.

Reply:
Done